spec/adapters/replication_transfer_adapter_spec.rb

```diff
@@ -17,6 +17,7 @@
   updated_at = "2015-02-25T15:27:40Z"
   created_at = "2015-02-25T15:27:40Z"
   link = "user@herp.derp.org:/blah"
+  cancel_reason_detail = "testing is fun # comment // woo"
```

Comment:
Could this move to a fabricator?  (Likewise for other instances in other spec files.)